src/modules/iotjs_module_buffer.c

```diff
@@ -82,6 +82,21 @@ size_t iotjs_bufferwrap_length(iotjs_bufferwrap_t* bufferwrap) {
 }
 
 
+bool iotjs_jbuffer_is_bufferwrap(const jerry_value_t jbuffer) {
```

Comment:
I would prefer something similar as `iotjs_bufferwrap_from_jbuffer` except it return NULL if not a buffer. That way we could avoid getting a property twice (property read is costly).